src/assets/items/items.ts

```diff
@@ -82,8 +82,15 @@ export const items: { [id: string]: Item } = {
         value: 250,
         description: "You lucky motherfucker",
         type: ItemType.material,  
+    },
+    MinecraftSteveDiamondPickaxe:{
```

Comment:
Make item key same as item Ids as it is used to index items many places. item Ids should also be all lowercase.